content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -1037,7 +1037,7 @@ Continue Token, Exact
 {{< note >}}
 When you **list** resources and receive a collection response, the response includes the
 [metadata](/docs/reference/generated/kubernetes-api/v1.21/#listmeta-v1-meta) of the collection as
```

Comment:
```suggestion
[metadata](/docs/reference/generated/kubernetes-api/v{{< skew currentVersion >}}/#listmeta-v1-meta) of the collection as
```

Reply:
@pranav-pandey0804
Does the suggestion make sense?

Reply:
@kbhawkey yes, the suggestion was to replace 'v1.21' with 'v{{< skew currentVersion >}}' so that the version gets assigned dynamically but as the change is to be made in the target URL, the replacement will not redirect the changed URL to the targeted page but rather give an error which might also result in another broken or invalid link.

Can you please suggest me about the [first commit](https://github.com/kubernetes/website/pull/40914/commits/a2ec0da8ec9e96bf2082eedd8305a1c6e0be8f24)

Reply:
>  the replacement will not redirect the changed URL to the targeted page

why is that?

Reply:
i think because the pages are generated statically. @reylejano could you please confirm this?

Reply:
> i think because the pages are generated statically. @reylejano could you please confirm this?

Yes. And we always generate them for each new release.

Reply:
the link on 1039 is supposed to be redirecting at 'list-v1-meta' whereas the link on line 1040 is supposed to be directing at object-v1-meta, the link on 1039 and 1040 both were redirecting at list-v1-meta, so i changed the one on 1040 so that it redirects us to the 'object-v1-meta'.

Reply:
Right. Then both 1039 and 1040 should use the shortcode to point to the "current" API version, right?

Reply:
thanks @tengqm  for the info, yes both lines are need to be updated as suggested .
I will update it accordingly.